packages/pyright-internal/src/tests/samples/semantic_highlighting/builtin_identifiers.py

```diff
@@ -0,0 +1,14 @@
+from typing import List, Set, TypeAlias
+
+Foo = list
+Bar = List | Set
+Old: TypeAlias = dict
+type New = tuple
+
+print
+input
+func = print
+
+int
+str
+ValueError
```

Comment:
Can we add `EnvironmentError` here, it's an alias that is defined in `builtins` as `EnvironmentError = OSError`, but if it's ever updated to `: TypeAlias`, then our implementation might break.

Reply:
Done :+1: